index.html

```diff
@@ -243,10 +244,91 @@ <h4 class="media__body__heading">Brenton Cleeland</h4>
                                     </ul>
                                 </div>
                             </div>
+                        </article> -->
+
+		      <article class="past-speaker Grid-cell">
```

Comment:
Can you fix the indentation here to be all spaces and not a mixture of spaces and tabs, please :)